### PR TITLE
Takeoff State Logic

### LINKED ISSUE #76: Takeoff State Logic
# Takeoff State Logic

## Problem

Need to arm the drone and takeoff to the desired operating altitude.

## Solution

Utilize the MAVSDK commands to have the drone takeoff from the starting position and ascend to the right takeoff altitude.

### PR DESCRIPTION
**Pull request by carson-williams**

### CODE REVIEW

File: flight/states/takeoff.py [hunk 1]
```diff
@@ -29,4 +31,18 @@ async def run(self, drone: System) -> Waypoints:
         Waypoints : State
             Next state to fly waypoint path
         """
+        # Initialize altitude (convert feet to meters)
+        await drone.action.set_takeoff_altitude(75 / 3.2808)
```
Comment (cjhr95) [inline flight/states/takeoff.py]: Here I would import the constant TAKEOFF_ALT from the config.py file here, so it doesn't seem like this value is appearing out of thin air.

**Review by cjhr95 — CHANGES_REQUESTED** [review]
Just a small thing about the takeoff altitude.

**Review by cjhr95 — APPROVED** [review]
Looks good!

**Comment by fallscameron01** [comment]
If this is good to merge, you can go ahead and do that @cjhr95

**Pull request merged**